src/JuliusSweetland.OptiKey.Core/Native/PInvoke.cs

```diff
@@ -120,5 +120,8 @@ public static long GetWindowLong(IntPtr hWnd, int nIndex)
 
         [DllImport("user32.dll")]
         public static extern void keybd_event(byte bVk, byte bScan, uint dwFlags, int dwExtraInfo);
+
```

Comment:
Added this to combine 4 individual redraws after each of the 4 adjustments: x, y, width, height into a single redraw.